src/sagas/index.js

```diff
@@ -88,8 +88,10 @@ const VM_FETCH_ADDITIONAL_SHALLOW = [
 export const EVERYONE_GROUP_ID = 'eee00000-0000-0000-0000-123456789eee'
 
 export function* transformAndPermitVm (vm) {
+  const entityPermissionsToUserPermits = yield curryEntityPermissionsToUserPermits()
```

Comment:
just to be consistent with other appearances of this `curryEntityPermissionsToUserPermits` call:
```suggestion
  const _entityPermissionsToUserPermits = yield curryEntityPermissionsToUserPermits()
```